runtime/jvmti/jvmtiThread.c

```diff
@@ -364,6 +364,11 @@ jvmtiStopThread(jvmtiEnv* env,
 
 		ENSURE_JOBJECT_NON_NULL(exception);
 
+		ENSURE_JTHREAD_NON_NULL(thread);
+#if JAVA_SPEC_VERSION >= 19
```

Comment:
dont think thes macros are needed since you are defining `ENSURE_JTHREAD_NOT_VIRTUAL` in preJDK19 code as well

Reply:
right, but youve defined `ENSURE_JTHREAD_NOT_VIRTUAL` to do nothing on pre jdk19

Reply:
If you are talking about https://github.com/eclipse-openj9/openj9/pull/15418/files#diff-5d7352c74c88b671804dc6ab968ad82f084f228182a671db384c234b09bc5a00R518 that is for `#ifdef J9_JVMTI_PERFORM_CHECKS` #else block (which I should have added if Java >= 19 wrappers for as well)

Reply:
ah okay